src/applications/proxy-rewrite/partials/desktop/header/index.js

```diff
@@ -0,0 +1,109 @@
+import { makeMegaMenu } from './mega-menu';
```

Comment:
Plain HTML version of the header (used for desktop) excluding the mega menu, which is in a different file